vscode/package.json

```
@@ -830,6 +846,12 @@
           "markdownDescription": "Enable OpenTelemetry tracing",
           "default": false
         },
+        "cody.experimental.commands": {
+          "order": 99,
+          "type": "boolean",
+          "markdownDescription": "Enable the updated generate unit tests command",
+          "default": false
+        },
```

```suggestion
        "cody.experimental.inlineUnitTest": {
          "order": 99,
          "type": "boolean",
          "markdownDescription": "Enable the updated generated unit tests command",
          "default": false
        },
```

nit: more specific, in case we have another `experimental.commands` flag.

Feel free to ignore this if we're going to quickly delete this flag soon

---

> Feel free to ignore this if we're going to quickly delete this flag soon

Yea since i'm planning to delete this flag soon, I will just hide it behind `internal.unstable` instead, thanks!